tests/support/createTestWidget.ts

```diff
@@ -1,7 +1,7 @@
-import { WidgetBase } from './../../src/WidgetBase';
-import { w } from './../../src/d';
-import { Registry } from './../../src/Registry';
-import { Constructor, VirtualDomNode, WidgetBaseInterface } from './../../src/interfaces';
+import { WidgetBase } from '../../src/WidgetBase';
```

Comment:
Is this something your IDE does automatically?

Reply:
My IDE (IntelliJ IDEA) doesn't like paths that start with "./../".  I could turn the checks off but what is the purpose of using "./.."?

Reply:
It's just my habit really (probably why it's all through the repo)